MM-33013 Custom status feature (#4330)
mattermost/docs: source/help/settings/account-settings.rst

To enable, download Google Authenticator from `iTunes <https://itunes.apple.com/us/app/google-authenticator/id388497605?mt=8>`__ or `Google Play <https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en>`__ for your phone, then:

1. Select the **Add MFA to your account** button.
2. Use Google Authenticator to scan the QR code that appears.

> 3. Type in the Token generated by Google Authenticator, then select **Save**.

Reviewer: ```suggestion
3. Enter the token generated by Google Authenticator, then select **Save**.
```

In future, you will be asked to enter a passcode from Google Authenticator in addition to your password on login.

If you are unable to scan the QR code, you can:

1. Select the **Add MFA to your account** button. You should see a ``Secret Key`` below the QR code.
2. Open Google Authenticator.
3. Select the **+** button, then select **Manual Entry**.
4. Enter an account name.
5. Enter the ``Secret Key`` from Mattermost into the ``Key`` field in Google Authenticator, then select **Save**.

> 6. In Mattermost, type the token generated by Google Authenticator, then select **Save**.

Reviewer: ```suggestion
6. In Mattermost, enter the token generated by Google Authenticator, then select **Save**.
```

Send Mobile Push Notifications
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Push notifications can be sent to your mobile device if you have the Android or iOS app installed. You can choose the type of activity that will send a notification. By default, push notifications are sent "For mentions and direct messages". If push notifications are sent "Never", the Mattermost setting to trigger push notifications depending on your Mattermost `availability <https://docs.mattermost.com/help/getting-started/setting-your-status-availability.html>`__ is hidden. If your System Admin has not set up push notifications, this setting will be disabled.

If you're actively viewing a channel (Public or Private), Direct Message, or Group Message on desktop or web, no push notifications will be sent for that channel. If a notification is viewed on desktop or web, the lockscreen notification will clear on Android, and on iOS the badge on the Mattermost app icon will decrement accordingly.

Trigger Mobile Push Notifications When
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> You can also choose when to send push notifications depending on your Mattermost availability. By default, push notifications are sent if your availability is "Away" or "Offline". If "Send Mobile Push Notifications" is set as "Never", this setting is hidden. If your System Admin has not set up push notifications, this setting will be disabled.

Reviewer: ```suggestion
You can also choose when to send push notifications depending on your Mattermost availability. By default, push notifications are sent if your availability is **Away** or **Offline**. If **Send Mobile Push Notifications** is set as **Never**, this setting is hidden. If your System Admin has not set up push notifications, this setting will be disabled.
```

Automatic Direct Message Replies
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Set a custom message that will be automatically sent in response to Direct Messages. Mentions in Public and Private Channels will not trigger the automated reply. Enabling Automatic Replies sets your availability to "Out of Office" and disables desktop, email, and push notifications. This setting is experimental and `must be enabled by your System Admin <https://docs.mattermost.com/administration/config-settings.html?highlight=config%20settings#enable-automatic-replies-experimental>`__.

Reviewer: ```suggestion
Set a custom message that will be automatically sent in response to Direct Messages. Mentions in Public and Private Channels will not trigger the automated reply. Enabling Automatic Replies sets your availability to **Out of Office** and disables desktop, email, and push notifications. This setting is experimental and `must be enabled by your System Admin <https://docs.mattermost.com/administration/config-settings.html?highlight=config%20settings#enable-automatic-replies-experimental>`__.
```
